gradle/plugins/build-parameters/build.gradle.kts

```diff
@@ -0,0 +1,85 @@
+plugins {
+	id("org.gradlex.build-parameters") version "1.4.3"
+}
+
+group = "junitbuild"
+
+buildParameters {
+	pluginId("junitbuild.build-parameters")
+	bool("ci") {
+		description.set("Whether or not this build is running a CI environment")
+		defaultValue.set(false)
+		fromEnvironment()
+	}
+	integer("javaToolchainVersion") {
+		description.set("Defines the Java Toolchain version to use for compiling code")
+	}
+	group("buildCache") {
+		string("username") {
+			description.set("Username to authenticate with the remote build cache")
+			fromEnvironment()
+		}
+		string("password") {
+			description.set("Password to authenticate with the remote build cache")
+			fromEnvironment()
+		}
+		string("url") {
+			description.set("URL to the remote build cache")
+			fromEnvironment()
+		}
+	}
+	group("documentation") {
+		description.set("Parameters controlling how the documentation is built")
+		bool("replaceCurrentDocs") {
+			description.set("The documentation that is being deployed will replace what's currently deployed as 'current'")
+			defaultValue.set(false)
+		}
+	}
+	group("enterprise") {
+		description.set("Parameters controlling Gradle Enterprise features")
+		string("accessKey") {
+			description.set("The access key used to authenticate with Gradle Enterprise")
+			fromEnvironment("GRADLE_ENTERPRISE_ACCESS_KEY")
+		}
```

Comment:
I'm on the fence about this one since it's value is not actually used to configure the access key, just for checking whether one is configured using the [official env variable](https://docs.gradle.com/enterprise/gradle-plugin/#via_environment_variable). I think it might be better to keep using `System.getenv` in this case. @britter WDYT?

Reply:
I've removed it now since it [failed the build](https://github.com/junit-team/junit5/actions/runs/4367464590/jobs/7638807425). Let's see if that resolves it.